src/styles/blocks/card.scss

```
@@ -0,0 +1,120 @@
+.card {
+  background-color: #fff;
+  width: $card-width;
+  padding: 32px 16px 16px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  transition: transform 0.3s;
+
+  &:hover {
+    transform: scale(1.2);
+  }
+
+  &__img {
+    display: block;
+    width: 160px ;
+    height: 134px ;
+    margin: 0 auto;
+    margin-bottom: 40px;
+
+  }
+
+  &__name {
+    font-weight: 500;
+    margin: 0;
+    font-size: 12px;
+    line-height: 18px;
+    margin-bottom: 4px;
+  }
+
+  &:hover .card__name {
+    color: #34568b;
+  }
+
+  &__code {
+    color: $secondary-color;
+    font-weight: 400;
+    margin: 0;
+    font-size: 10px;
+    line-height: 14px;
+    margin-bottom: 16px;
+  }
+
+  &__price {
+    font-weight: 400;
+    font-size: 12px;
+    line-height: 18px;
+    color: $secondary-color;
+  }
+
+  &__value {
+    font-weight: 700;
+    font-size: 16px;
+    line-height: 18px;
+    color: $main-color;
+  }
+
+  &__buy {
+    width: 166px;
+    height: 40px;
+    display: flex;
+    justify-content: center;
+    font-weight: 700;
+    font-size: 14px;
+    line-height: 40px;
+    text-align: center;
+    text-transform: uppercase;
+    text-decoration: none;
+
+    background-color: $blue-color;
+    color: white;
+    border-radius: 5px;
+    border: 1px solid $blue-color;
+  }
+
+  &__buy:hover {
+    background-color: #fff;
+    color: $blue-color;
+  }
+
+  &__price-box {
+    display: flex;
+    justify-content: space-between;
+    margin-bottom: 16px;
+  }
+
+  &__review-box {
+    height: 16px;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin-bottom: 24px;
+  }
+}
+
+.reviews {
```

it should be as an element of card block. Move it inside .card selector
```suggestion
&__reviews {
```